salt/tests/unit/modules/conftest.py

```diff
@@ -0,0 +1,11 @@
+import os.path
+import sys
+
+# Add our Salt module directory to the python path
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))),
+        "_modules"
+    )
```

Comment:
Could be easier (IMHO) to read with:

```suggestion
    os.path.realpath(os.path.join(*[
        __file__, "..", "..", "..", "..", "_modules"
    ]))
```

Reply:
It will be more something like this (since `__file__` is a file and not a directory)
```suggestion
    os.path.realpath(os.path.join(
        os.path.dirname(__file__), "..", "..", "..", "_modules"
    ))
```
Personally prefer the current approach ...

Reply:
Fine by me, anyway as soon as we get Python 3 we'll rewrite it with `pathlib` :stuck_out_tongue_winking_eye: